README.md

```diff
@@ -1,15 +1,13 @@
 # Harbor
 
-[![CI](https://github.com/goharbor/harbor/workflows/CI/badge.svg?branch=main&event=push)](https://github.com/goharbor/harbor/actions?query=event%3Apush+branch%3Amain+workflow%3ACI+)
```

Comment:
I think we should not remove the CI badge, but the original link is incorrect, we can replace it to `https://github.com/goharbor/harbor/actions/workflows/CI.yml/badge.svg?branch=main`.

Reply:
Agree that we should fix it rather than remove it.